src/lashup_kv_sync_rx_fsm.erl

```diff
@@ -89,11 +89,13 @@ handle_disconnect({'DOWN', _MonitorRef, _Type, _Object, Reason}) ->
 init_metrics() ->
     prometheus_counter:new([
         {registry, lashup},
-        {name, aae_rx_bytes_total},
-        {help, "Total number of AAE RX bytes received by node."}
+        {name, aae_rx_messages_total},
+        {help, "Total number of AAE (Active Anti-Entropy) RX messages "
```

Comment:
Please see this discussion https://github.com/dcos/lashup/pull/49#discussion_r264419700. I think that it's quite hard to understand any lashup metrics if you don't know basic stuff about crdt-based distributed databases and  how lashup works internally. It's even hard to understand the metrics if you know how lashup works. We can write some documentation about AAE if it's needed. It just don't make much sense to transcribe widely used acronyms in help strings.

Reply:
I'd argue that AAE is a widely used acronym, is something new to me. A bit of a story.

- I encountered this term multiple times since, I believe, 2012.
- 11 months ago, shortly after joining I read Lashup code, and back then I managed to find out what AAE stood for on my own.
- A few days back I failed to do so, I guess, because it didn't occur to me to check Riak docs, like it did back then, and spend a couple of dozens of minutes grepping Lashup, checking the papers it is based upon and so forth, without any luck. And only then I resorted to asking in the Slack channel.

I'd say it's worth it because at the very least, quite probably, it'll spare me a few minutes next time.